front_end/css/profile-styles.css

```diff
@@ -52,4 +52,12 @@
 
 .bottom-spacing {
     margin-bottom: 40px;
+}
+
+#request-table {
```

Comment:
If you think we might want this code in the future, it's fine to leave it here commented. If not, please remove it. If you do keep it, please leave a comment on what it does so that we will know in the future what it does.

Reply:
This isn't commented out. When a style starts with a #, that means it's connected to an id rather than a class.